tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     py{27,36}-sphinx{15,16,17,18}
-    py{36,37,38,39}-sphinx{21,22,23,24,30,31,32,33,34,latest}
+    py{36,37,38,39,310}-sphinx{21,22,23,24,30,31,32,33,34,35,40,41,42,43,44,latest}
```

Comment:
Looks like sphinx 3.5, 4.0 and 4.1 aren't compatible with python 3.10 (bc of changes on the typing module). Thoughts about testing 3.10 only for sphinx>=4.2?

Reply:
Sounds good to me.